composer.json

```diff
@@ -38,6 +38,7 @@
         "psr-4": {
             "Rector\\": "src",
             "Rector\\BetterReflection\\": "packages/BetterReflection/src",
```

Comment:
Btw, the https://github.com/rectorphp/BetterReflection package does not need to exist anymore, doesn't? As we have it build in Rector?

Reply:
It does. It only fork of better reflection package with php-parser 4.0 support.

The one in here in `/packages` is few helper classes over it.